packages/components/tooltip/src/tooltip.vue

```diff
@@ -207,6 +210,10 @@ export default defineComponent({
       return popperContent && popperContent.contains(document.activeElement)
     }
 
+    const isActive = ref(true)
+    onActivated(() => (isActive.value = true))
+    onDeactivated(() => (isActive.value = false))
```

Comment:
It seems that you only need to judge whether the content exists here, and then call `hide` according to the situation.

Reply:
Thank you very much for your ideas。
Component open property is now controlled by the keep-alive hook function